infra/examples/gcp-google-workspace/main.tf

```
@@ -115,8 +144,83 @@ module "worklytics-psoxy-connection" {
   for_each = module.worklytics_connector_specs.enabled_google_workspace_connectors
 
   # source = "../../modules/worklytics-psoxy-connection"
-  source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-psoxy-connection?ref=v0.4.2"
+  source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-psoxy-connection?ref=v0.4.3"
 
   psoxy_endpoint_url = module.psoxy-google-workspace-connector[each.key].cloud_function_url
-  display_name       = "${each.value.display_name} via Psoxy"
+  display_name       = "${title(each.key)}${var.connector_display_name_suffix} via Psoxy"
+}
+
+# BEGIN LONG ACCESS AUTH CONNECTORS
+
+resource "google_service_account" "long_auth_connector_sa" {
+  for_each = module.worklytics_connector_specs.enabled_oauth_long_access_connectors
+
+  project      = google_project.psoxy-project.project_id
+  account_id   = "psoxy-${each.key}"
```

seen this limit in the example. Are SA names limited to 30 chars?
```suggestion
  account_id   = "psoxy-${substr(each.key, 0, 24)}"
```